frontend/src/pages/modelServing/useModelMetricsEnabled.ts

```diff
@@ -0,0 +1,14 @@
+import { useAppContext } from 'app/AppContext';
+import { useDashboardNamespace } from 'redux/selectors';
+import { isModelMetricsEnabled } from './screens/metrics/utils';
+
+const useModelMetricsEnabled = (): { modelMetricsEnabled: () => boolean } => {
+  const { dashboardNamespace } = useDashboardNamespace();
+  const { dashboardConfig } = useAppContext();
+
+  const modelMetricsEnabled = () => isModelMetricsEnabled(dashboardNamespace, dashboardConfig);
+
+  return { modelMetricsEnabled };
```

Comment:
Why not return the boolean?

Reply:
Does it matter? I can change it if you want, no problem!

Reply:
It's a weird pattern, you cause it to be lazy for no reason, it's not "wrong" -- just "weird".

Not quite sure why you did it this way, so I asked the question. It's fine, leave it.

Reply:
ok! no, no worries, i was asking cause i was in other pr, thanks for the clarification, changing it right now.